server/game/effects.js

```diff
@@ -122,6 +122,8 @@ const Effects = {
     restrictHouseChoice: (house) => EffectBuilder.player.static('restrictHouseChoice', house),
     stealFromPool: () => EffectBuilder.player.static('stealFromPool'),
     captureFromPool: () => EffectBuilder.player.static('captureFromPool'),
+    captureOneMoreFromPool: () => EffectBuilder.player.static('captureOneMoreFromPool'),
```

Comment:
I believe the implementation is good, but we could make this more flexible by adding an amount. You could name, for example: "captureMoreFromPool" or "captureExtraFromPool"

Reply:
Good suggestion, done!